CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Primary Composite Key - Primary Composite Key are supported by the liana.
```

Comment:
no need to specify `by the liana`, developers that will read the changelog know that it is related to the current liana.